dnf/cli/option_parser.py

```diff
@@ -58,6 +58,13 @@ def __call__(self, parser, namespace, values, opt_str):
             l = getattr(namespace, self.dest)
             l.extend((x, operation) for x in re.split(r'\s*[,\s]\s*', values))
 
+    class _RepoCallbackEnable(argparse.Action):
+        def __call__(self, parser, namespace, values, opt_str):
+            operation = 'enable'
+            l = getattr(namespace, 'repos_ed')
```

Comment:
No, it does't work:
  File "/usr/lib/python3.5/site-packages/dnf/cli/option_parser.py", line 65, in __call__
    namespace['repos_ed'].append((values[0], operation))
TypeError: 'Namespace' object is not subscriptable